gradlePlugin/src/main/kotlin/org/jetbrains/kotlinx/dl/gradle/ModelType.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2022 JetBrains s.r.o. and Kotlin Deep Learning project contributors. All Rights Reserved.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlinx.dl.gradle
+
+private const val ROOT_FOLDER = "models/onnx/mobile"
+private const val SUFFIX = "ort"
```

Comment:
Maybe it's worth supporting both '.ort' and '.onnx' here

Reply:
Do you mean to support this task for desktop as well, or to support onnx models for android?

Reply:
I am talking just about the file extension. We support both .ort and .onnx formats on android, but here it's supposed that only '.ort' is possible. On the other hand, we can modify it later

Reply:
No, you are right, let's support different extensions.